generator-jsonforms/src/generators/app/index.test.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Test cases for the Generators package.
+ */
+describe('Generators package tests', () => {
```

Comment:
Any plans on actually testing something here (applies to other tests as well)?

Reply:
Until the final presentation we just set up the tests and could not complete them.